Documentation/tutorial/docfx.exe_user_manual.md

```diff
@@ -158,6 +158,8 @@ docfx pdf [<config_file_path>] [-o:<output_path>]
 
 Current design is that each TOC file generates a corresponding PDF file. Walk through [Walkthrough: Generate PDF Files](walkthrough/walkthrough_generate_pdf.md) to get start.
 
+If `cover.md` is found in a folder, it will be rendered as the cover page.
```

Comment:
This document may also need update:
https://dotnet.github.io/docfx/tutorial/walkthrough/walkthrough_generate_pdf.html#step1-add-a-tocyml-specific-for-pdf

> TOC is also used as the cover page of the PDF